justfile

```diff
@@ -19,11 +19,10 @@ test-unit:
 bootstrap:
     go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.32
     go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.3
-    go install github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen@v2.3
+    go install github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen@latest
     go install golang.org/x/tools/cmd/godoc@latest
 gen:
```

Comment:
`just gen` now requires `just bootstrap` so the script has the right dependencies to call... should it call `bootstrap` itself or is unexpectedly installing dependencies weird?

Reply:
yeah, that would be weird. `bootstrap` shouldn't be a requirement for `just gen`